substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/SerializationConfigurationParser.java

```diff
@@ -39,6 +39,7 @@
 public class SerializationConfigurationParser extends ConfigurationParser {
```

Comment:
If I understand this code correctly, the new format doesn't make sense to me and supporting a new attribute with the old format is unnecessary too. What I had in mind is something like this, where you can also easily distinguish between the old and new formats because with the new one, the top level is a set instead of an array:
```json
{
  "types": [
    { "name": "com.oracle.Foo" },
    { "name": "com.oracle.Bar", "customTargetConstructorClass": "com.oracle.FooBar" }
  ],
  "capturingTypes": [
    { "name": "com.oracle.Baz" }
  ]
}
```

Reply:
Fixed as suggested.

Reply:
But it is only for lambdas, so just `capturingTypes` is not correct?